include/head/core/presence_sensing_driver.hpp

```diff
@@ -5,22 +5,25 @@
 
 #include "can/core/ids.hpp"
 #include "common/core/bit_utils.hpp"
-#include "head/core/adc.hpp"
 #include "head/core/attached_tools.hpp"
 
 namespace presence_sensing_driver {
 using namespace can::ids;
 
 class PresenceSensingDriver {
   public:
-    explicit PresenceSensingDriver(adc::BaseADC& adc)
-        : PresenceSensingDriver(adc, attached_tools::AttachedTools{}) {}
-    PresenceSensingDriver(adc::BaseADC& adc,
-                          attached_tools::AttachedTools current_tools)
-        : adc_comms(adc), current_tools(current_tools) {}
-    auto get_readings() -> adc::MillivoltsReadings {
-        return adc_comms.get_voltages();
-    }
+    PresenceSensingDriver()
+        : PresenceSensingDriver(attached_tools::AttachedTools{}) {}
+    explicit PresenceSensingDriver(const PresenceSensingDriver&) = default;
+    auto operator=(const PresenceSensingDriver&)
```

Comment:
Do we want this class to be moveable/copy-able?

Reply:
it doesn't really matter IMO it doesn't capture any references to things